.github/ISSUE_TEMPLATE/release-todo-issue-template.yml

```
@@ -10,40 +10,42 @@ body:
     description: Complete the following items to perform a release.
     options:
+    - label: "Title updated by replacing the '**_<TODO: release type here>_**' section with the type of release."
+    - label: "Title updated by replacing the '**_<TODO: release version number here>_**' section with the version number."
+    - label: Appropriate release label added to this issue.  Refer to the _**Release Labels**_ section below.
+    - label: Release PR linked to this issue.
     - label: Manual QA Testing completed.
-    - label: Title updated by replacing the '**_\<release type here\>_**' section with the type of release.
-    - label: Title updated by replacing the '**_\<release number here\>_**' section with the release number.
-    - label: Issues listed in the **_Issues Released_** section above for all issues being released.
-    - label: All issues in the _**Issues Released**_ section are closed and all tasks are complete.
-    - label: Appropriate release label added to this issue.  Refer to the release label section below.
-    - label: Appropriate non-release labels added to reflect the type of changes being released.   Refer to the type of changes section below.
-    - label: Correct release PR linked to this issue.
+    - label: All issues listed in the **_Issues Released_** section above.
+    - label: All issues in the _**Issues Released**_ section are closed and all issue tasks are complete.
     - label: All unit tests have been executed locally and have passed. (Check out the appropriate release branch before running tests).
-    - label: Version in project file updated.
-    - label: Release notes created and added.
-    - label: All changes made directly on the release branch.
-    - label: Manual release to **_preview_** or **_production_** executed depending on the type of release.  Refer to the release execution section below.
+    - label: Version in project file updated (All changes made directly on the release branch).
+    - label: Release notes created and added (All changes made directly on the release branch).
+    - label: Release to **_preview_** or **_production_** complete.  Refer to the release execution section below.
 
 
 # POST-RELEASE TODO LIST
 - type: checkboxes
   attributes:
-    label: Post-Release To Do List
+    label: Post-Release ToDo List
     description: Verify that release went smoothly.
     options:
-    - label: The release displayed in [GitHub Releases](https://github.com/KinsonDigital/[TODO: add project name here]/releases) is correct.    
-    - label: The [nuget package](https://www.nuget.org/packages/KinsonDigital.[TODO: add project name here]/) has been successfully deployed to the public repository.
+    - label: The release displayed in [GitHub Releases](https://github.com/KinsonDigital/Velaptor/releases) is correct.    
+    - label: The [nuget package](https://www.nuget.org/packages/KinsonDigital.Velaptor/) has been successfully deployed to the public repository.
```

Please fix URL's. They used to have the TODO instead of the project name, but when you replaced the entire template with Velaptor's, you forgot to fix the URL's to show the todo items.

```suggestion
    - label: The release displayed in [GitHub Releases](https://github.com/KinsonDigital/[TODO: add project name here]/releases) is correct.    
    - label: The [nuget package](https://www.nuget.org/packages/KinsonDigital.[TODO: add project name here]/) has been successfully deployed to the public repository.
```